readme.md

```diff
@@ -52,7 +52,7 @@ this.sendNotification("MAX_PAGES_CHANGED", 4);
 
 This would now show that there are now 4 pages to display.
 
-You can also just send `PAGE_INCREMENT` or `PAGE_DECREMENT` without any payloads (or with, but it will be ignored) to have the module change the displayed page by one.
+You can also just send `PAGE_INCREMENT` or `PAGE_DECREMENT` without any payloads to have the module change the displayed page by one. If you are using [MMM-pages](https://github.com/edward-shen/MMM-pages.git) with this module, sending an integer as a payload for a `PAGE_INCREMENT` or `PAGE_DECREMENT` notification will perform as described in [the MMM-pages documentation](https://github.com/edward-shen/MMM-pages#notifications). If you are not using MMM-pages, the indicator will ignore the payload and just change the displayed page by one.
```

Comment:
I thought the only changes would be that the `PAGE_INCREMENT` and `PAGE_DECREMENT` notifications would be ignored if MMM-pages was detected? This description seems to imply that the payload to these notifications would be conditionally handled, which doesn't seem correct?

Reply:
Yeah, so I didn't want to break compatibility for people who may be using this module without MMM-pages, so I made the notification handler only ignore the notifications on the condition that MMM-pages is detected. If it is not detected, then it will react to `PAGE_INCREMENT` and `PAGE_DECREMENT` notifications, but with MMM-page-indicator's existing behavior of ignoring the payload and just changing the page by one in the respective direction. 

The description is also trying to describe how MMM-pages and MMM-page-indicator work with each other as other modules broadcast `PAGE_INCREMENT` and `PAGE_DECREMENT` notifications. With the changes made in this PR, if it detects MMM-pages, MMM-page-indicator will ignore the `PAGE_INCREMENT` and `PAGE_DECREMENT` notifications and simply act as a consumer to the `NEW_PAGE` notifications broadcasted by MMM-pages, since MMM-pages handles `PAGE_INCREMENT` and `PAGE_DECREMENT` notifications itself (which can be seen [here](https://github.com/edward-shen/MMM-pages/blob/a529913f4067c2608fe4d6c70cc2bbc25e34415c/MMM-pages.js#L102)) and sends `NEW_PAGE` notifications to let other modules know which page is being shown. Since MMM-pages can handle payloads of `number` type, I thought it was worth mentioning that the user can send these payloads along with a `PAGE_INCREMENT` and `PAGE_DECREMENT` notification and the indicator would update correctly.